### PR TITLE
feat(Root): fix padding in demo table

### PR DESCRIPTION
**Pull request by xaviermariaconejo**
1) Fix padding in demo tables

2) We have noticed that some components are using the user agent styles, which is a bug. But since we should have deployed that studio visually correct, we have removed the reset styles. There still is the problem that the components that are using the user agent styles have bug, and we should try to all the different frontals use the same reset styles.

### CODE REVIEW

**Review by adriadoz — APPROVED** [review]

File: demo/package.json [hunk 1]
```diff
@@ -8,5 +8,8 @@
       "packagesFolder": ".",
       "deepLevel": 3
     }
+  },
+  "dependencies": {
+    "undefined": "^0.1.0"
```
Comment (oriolpuig) [inline demo/package.json]: Could you remove that dependency? As far as I know, it's a known issue at sui-mono.

File: demo/package.json [hunk 2]
```diff
@@ -8,5 +8,8 @@
       "packagesFolder": ".",
       "deepLevel": 3
     }
+  },
+  "dependencies": {
```
Comment (midudev) [inline demo/package.json]: Remove this.

File: package.json [hunk 3]
```diff
@@ -30,11 +31,11 @@
     "@s-ui/mono": "1",
     "@s-ui/precommit": "2",
     "@s-ui/studio": "8",
-    "fs-extra": "8.1.0",
-    "globby": "10.0.1",
-    "husky": "0.13.4",
-    "validate-commit-msg": "2.12.2",
-    "walker": "1.0.7"
+    "fs-extra": "8",
```
Comment (oriolpuig) [inline package.json]: Why we have updated the version pointing the major one? Normally we do that only on our private packages or SUI packages.

**Review by oriolpuig — APPROVED** [review]

File: package.json (hunk 3 above)
Comment (midudev) [inline package.json]: We don't rely on major versions for external dependencies so, just use the fixed versions.

File: demo/package.json (hunk 1 above)
Comment (xaviermariaconejo) [reply demo/package.json]: Yes, sorry

File: package.json (hunk 3 above)
Comment (xaviermariaconejo) [reply package.json]: Removed!

Comment (xaviermariaconejo) [reply package.json]: Fixed!

File: demo/package.json (hunk 2 above)
Comment (xaviermariaconejo) [reply demo/package.json]: done

**Review by midudev — APPROVED** [review]

**Pull request merged**